user_manual/pim/sync_android.rst

```diff
@@ -1,11 +1,11 @@
-===================
-Synchronize Android
-===================
+==========================
+Synchronizing with Android
+==========================
 
-Address book and Calendar
--------------------------
+Contacts and Calendar
+---------------------
 
-#. Install DAVdroid (possibly through F-Droid)
+#. Install `DAVdroid <https://www.davdroid.com/download/>`_ on your Android device.
```

Comment:
DavDroid has been renamed: https://www.davx5.com

Reply:
Ah ... Good to know. But nothing has changed besides the name, right?